sdk/eventhub/azure-eventhub/README.md

```diff
@@ -412,12 +412,41 @@ Reference documentation is available [here](https://azuresdkdocs.blob.core.windo
 The EventHubs SDK integrates nicely with the [Schema Registry][schemaregistry_service] service and [Avro][avro].
 For more information, please refer to [Schema Registry SDK][schemaregistry_repo] and [Schema Registry Avro Encoder SDK][schemaregistry_avroencoder_repo].
 
+### Pure Python AMQP Transport and Backward Compatibility
+
+The Azure Event Hubs client library is now based on a pure Python AMQP implementation. `uAMQP` has been removed as required dependency.
+
+To opt for `uAMQP` as the underlying transport:
+
+1. Install `uamqp` with pip.
+
+```
+$ pip install uamqp 
+```
+
+2. Pass `uamqp_transport=True` during client construction.
```

Comment:
Should we add in a sentence to say this is the only place where they have to do this ? all ops will happen from uamqp  or update the sample below 

Perhaps its clear given the section this is in

Reply:
hmm, I feel like it's clear given the numbered instructions. Thoughts @l0lawrence?